tests/pipeline/test_course_daily_metrics.py

```diff
@@ -141,16 +163,20 @@ def setup(self, db):
             ) for i, days in enumerate(self.cert_days_to_complete)]
 
     def test_get_enrolled_in_exclude_admins(self):
-
+        course_enrollments = CourseEnrollment.objects.filter(
+            course_id=self.course_overview.id)
         # Get the total number of course enrollments for the course
-        ce_count = CourseEnrollment.objects.filter(
-            course_id=self.course_overview.id).count()
+        ce_count = course_enrollments.count()
         # Get course admins (non-students) count for the course
-        ce_non_students = CourseAccessRole.objects.filter(
-            course_id=self.course_overview.id).count()
+        ce_students = course_enrollments.filter(
+            ~Q(user__courseaccessrole__role='course_creator_group'),
+            user__edly_profile__edly_sub_organizations=self.site.edly_sub_org_for_lms,
+            user__is_staff=False,
+            user__is_superuser=False,
+        ).count()
 
-        expected_count = ce_count - ce_non_students
-        assert ce_count > 0 and ce_non_students > 0 and expected_count > 0, 'say something'
+        expected_count = ce_students
+        assert ce_count > 0 and ce_students > 0 and expected_count > 0, 'say something'
```

Comment:
Lets update `say something` to more appropriate message.

Reply:
done